### PR TITLE
Update and standardize maven dependencies

### PR DESCRIPTION
**Pull request by loosebazooka**
deals with #2685 #2686

### CODE REVIEW

File: jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SettingsFixture.java [hunk 1]
```diff
@@ -67,18 +67,13 @@ static SettingsDecrypter newSettingsDecrypter(Path settingsSecurityFile) {
...
       setField(DefaultSecDispatcher.class, injectedDispatcher, "_cipher", injectCypher);
 
-      DefaultSettingsDecrypter settingsDecrypter = new DefaultSettingsDecrypter();
-      setField(
-          DefaultSettingsDecrypter.class,
-          settingsDecrypter,
-          "securityDispatcher",
-          injectedDispatcher);
-      return settingsDecrypter;
+      return new DefaultSettingsDecrypter(injectedDispatcher);
```
Comment (loosebazooka) [inline jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SettingsFixture.java:70]: signature changed, injection via reflection no longer necessary.

Comment (chanseokoh) [reply jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SettingsFixture.java:70]: Any chance `DefaultSecDispatcher` similarly allows setting `injectCypher` without using `setField`?

**Comment by chanseokoh** [comment]
We should not forget updating Maven on `jib-extensions`.

File: jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SettingsFixture.java (hunk 1 above)
Comment (loosebazooka) [reply jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SettingsFixture.java:70]: No, I checked that, it still requires some reflective injection :(

**Pull request merged**